### DISCUSSION TITLE
Why does the SuperiorSkyblock plugin crash after turning on the server?

### DISCUSSION THREAD

**Discussion by sktom1314** [BG-Software-LLC/SuperiorSkyblock2#1272] [Denied Feature Requests] [answered]
[crash-2022-07-15_22.18.58-server.txt](https://github.com/BG-Software-LLC/SuperiorSkyblock2/files/9126955/crash-2022-07-15_22.18.58-server.txt)

**Comment by OmerBenGera** [comment, marked as answer]
Open an issue on github and include your logs in there - make sure you upload them via pastebin
